pkgs/development/tools/dprint/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, fetchCrate, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dprint";
+  version = "0.15.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-1DUGp+HiiY03fyZ+b8hNUBIfuQV5Z/gEcOxc/vG3YiA=";
+  };
+
+  cargoSha256 = "sha256-twFXA8A+vP1n6IFJO78fKNs+FC2ui46rj1JmJ/eq3wc=";
+
+  # Tests fail because they expect a test WASM plugin. Tests already run for
+  # every commit upstream on GitHub Actions
```

Comment:
It is still nice to know if the program works in the nix environment, too.

Reply:
Hi, I've tried fixing this, but the tests [depend on the structure of the target directory too ](https://github.com/dprint/dprint/blob/d19785a19737058659f794bbcdbe4771d8dfdabf/crates/dprint/src/cli/run_cli.rs#L2436)